Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,801 @@
+                               <repository name="phyml" owner="gandres" />
+                            […]
+                            </repositories>
+                            ```
+                            - « toolshed » and « changeset_revision » are optional **but they are completed during upload on the toolshed**
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            <img src="../images/toolsuite3.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            ```
+                            <?xml version="1.0"?>
+                            <repositories description="Pipeline phylogeny">
+                               <repository name="mafft" owner="gandres" />
+                            </repositories>
+                            ```
+                            <img src="../images/toToolshed.png" class="no-border" >
+                            ```
+                            <?xml version="1.0"?>
+                            <repositories description="Pipeline phylogeny">
+                               <repository name="mafft" owner="gandres" toolshed="http://testtoolshed.g2.bx.psu.edu" changeset_revision="a895633568"/>
+                            </repositories>
+                            ```
+                        </script>
+                    </Section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            - New tool version
+                              - Latest version is no longer **changeset_revision="a895633568**
+                              - Reload `repository_dependencies.xml` file in your repository
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### 3.3 Workflow
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Workflow
+                            <img src="../images/workflow.png" class="no-border" >
+                            - How to order tools ?
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Workflow
+                            - In galaxy
+                            <img src="../images/galaxyWorkflowEx.png" class="no-border" >
```

Comment:
It should be great to have the same tools as the previous slide mafft, trimal ...